### ISSUE TITLE
Remove `role="banner"` from header component

### ISSUE THREAD

**Issue by bowrolio** [govuk-one-login/service-header#49]
The banner role is unnecessary for element header and produces warning messages on accessibility / HTML compliance tools. The role attribute should be removed.

See https://validator.w3.org/
